src/react-query/queryClient.ts

```diff
@@ -1,7 +1,31 @@
 import { QueryClient } from '@tanstack/react-query';
+import AsyncStorage from '@react-native-async-storage/async-storage';
+import { createAsyncStoragePersister } from '@tanstack/query-async-storage-persister';
+import { PersistQueryClientOptions } from '@tanstack/react-query-persist-client';
 
 export const queryClient = new QueryClient({
   defaultOptions: {
-    queries: {},
+    queries: {
+      cacheTime: Infinity,
```

Comment:
Yessss. For others' reference: https://www.notion.so/rainbowdotme/Async-State-RFC-711800896f9a458889fc4ffad4de271a#41a61b0c4ed24281bea7f8360c97f56a